bigbluebutton-html5/imports/api/common/server/helpers.js

```diff
@@ -45,3 +46,21 @@ export const extractCredentials = (credentials) => {
   const requesterUserId = credentialsArray[1];
   return { meetingId, requesterUserId };
 };
+
+export const publicationSafeGuard = function (fn, self) {
```

Comment:
Please add block 2-3 sentences comment above the function - what it does and why.
Every one second we check whether certain condition is still met. This is a way to monitor [....] and react in order to support X functionality